CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- **CUMULUS-1459**
```

Comment:
We need to call out an optional migration step of indexing from the database to pull the async operations in

Reply:
More like at the top: If you want any existing async operations to show up on your dashboard you'll need to perform an index from database function <link to docs>